src/reducers/index.js

```diff
@@ -0,0 +1,55 @@
+import { REQUEST_MOVIES, RECEIVE_MOVIES, FILTER_MOVIES, FILTER_MOVIES_BY, SORT_MOVIES } from '../actions';
+import { combineReducers } from 'redux';
+
+export const movies = (state = {movieData: []}, action) => {
+
+    switch(action.type) {
+        case REQUEST_MOVIES:
+            return { ...state }
```

Comment:
1. `state` is not modified, so it should be returned directly: `return state;`
2. If `REQUEST_MOVIES` action has no effect on state, is it really needed?